hooks/_common.sh

```diff
@@ -359,68 +359,33 @@ function common::terraform_init {
     TF_INIT_ARGS+=("-no-color")
   fi
 
-  if [ ! -d .terraform/modules ] || [ ! -d .terraform/providers ]; then
+  test ! -d .terraform/modules
+  recreate_modules=$?
+  test ! -d .terraform/providers
+  recreate_providers=$?
+
+  if [ $recreate_modules ] || [ $recreate_providers ]; then
```

Comment:
Shell has no boolean type of variables. Non-empty var always evaluate to true. Hence:
```suggestion
  recreate_modules=$([[ -d .terraform/modules ]] && echo true || echo false)
  recreate_providers=$([[ -d .terraform/providers ]] && echo true || echo false)

  if [[ $recreate_modules == false || $recreate_providers == false ]]; then
```

ps: adjust `true` and `false` values per the use case as I'm a bit confused on when the `recreate_*` should be true or false.